benchmarks/src/main/java/org/apache/druid/benchmark/FloatCompressionBenchmark.java

```diff
@@ -55,12 +55,15 @@
 @OutputTimeUnit(TimeUnit.MILLISECONDS)
 public class FloatCompressionBenchmark
 {
+  @SuppressWarnings("SSBasedInspection")
```

Comment:
Please try to minimize these suppressions. In most *Benchmark and *Test classes static fields could be converted into instance fields.

In classes where this cannot be done, it makes sense to add `@SuppressWarnings("SSBasedInspection")` once on the class declaration instead of separately on every field.

In any case, *each* `@SuppressWarnings("SSBasedInspection")` should have a comment after it specifying the concrete type of Structural Search inspection being suppressed. In this PR, many of them should all look like
```java
@SuppressWarnings("SSBasedInspection") // static field(s) cannot be final because set in an initializer block
```